crates/shadowsocks-service/src/local/redir/sys/unix/mod.rs

```diff
@@ -15,7 +15,7 @@ cfg_if! {
         #[allow(dead_code, non_upper_case_globals, non_snake_case, non_camel_case_types)]
         #[allow(clippy::useless_transmute, clippy::too_many_arguments, clippy::unnecessary_cast)]
         mod pfvar;
-    } else if #[cfg(target_os = "freebsd")] {
+    } else if #[cfg(any(target_os = "freebsd", target_os = "openbsd"))] {
         #[path = "pfvar_bindgen_freebsd.rs"]
```

Comment:
`pfvar_bindgen_freebsd.rs` is generated on FreeBSD. I suggest you to generate one on OpenBSD with [bindgen-cli](https://crates.io/crates/bindgen-cli).

Reply:
Ah actually, for bindany, I just reversed the comment-out of the old existing code. Maybe that's not working correctly? I'll look into it.

Reply:
WRONG COMMENT. :)

Reply:
Sorry I'm not familiar with this tool, but how can I do this? Is there a commandline for generating this?

Reply:
Is `pfvar.h` in OpenBSD included in release? If no, you should first download it from its source.

Create a header file, that only has `#include <pfvar.h>`.

```bash
$ cargo install bindgen-cli
$ bindgen <your_created_header_file> -o pfvar_bindgen_openbsd.rs
```

More reference: https://rust-lang.github.io/rust-bindgen/command-line-usage.html